package.json

```diff
@@ -15,9 +15,9 @@
     },
     "scripts": {
         "test": "jest DFSOnBST LRU LinearSearchList BinarySearchList TwoCrystalBalls BubbleSort DoublyLinkedList Queue Stack ArrayList MazeSolver QuickSort BTPreOrder BTInOrder BTPostOrder BTBFS CompareBinaryTrees DFSOnBST DFSGraphList Trie BFSGraphMatrix Map MinHeap",
-        "clear": "./scripts/clear",
+        "clear": "node ./scripts/clear.js",
         "prettier": "prettier --write ./src",
-        "generate": "./scripts/generate",
-        "day": "echo /home/mpaulson/personal/kata-machine/src/day2"
+        "generate": "node ./scripts/generate.js",
+        "day": "echo C:\\Users\\bruce\\learning\\kata-machine\\src\\day1"
```

Comment:
could you do something with this line?  everything is great except for your user name here :)

if you could make it relative, that would be great (drop C:\\Users\\bruce\\learning\\kata-machine\\)

Reply:
I have updated the PR. Is this how the output supposed to look? @ThePrimeagen 

![image](https://user-images.githubusercontent.com/91245898/190912292-d162af4d-b8ca-4fa4-a89e-1f0b50de8e88.png)